src/System.Diagnostics.Debug/tests/DebugTests.cs

```diff
@@ -9,6 +9,7 @@ namespace System.Diagnostics.Tests
     public class DebugTests
     {
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Net46)]
         public void Asserts()
         {
             VerifyLogged(() => { Debug.Assert(true); }, "");
```

Comment:
We should consider changing the project then to be netcoreapp1.0 specific and add another flavor that is net46 specific so we can run the same tests but the one library will setup the WindowsDebugLogger and the other doesn't. That allows us to still use the same test to verify both. 

Rant: This is why I don't like InternalsVisibleTo :(